crates/cheatnet/tests/contracts/src/common/constructor_simple2.cairo

```diff
@@ -10,15 +16,17 @@ mod ConstructorSimple2 {
         self.number.write(number + number2);
     }
 
-    #[external(v0)]
-    fn add_to_number(ref self: ContractState, number: felt252) -> felt252 {
-        let new_number = self.number.read() + number;
-        self.number.write(new_number);
-        new_number
-    }
+    #[abi(embed_v0)]
+    impl ConstructorSimple2Impl of super::IConstructorSimple2<ContractState> {
+        fn add_to_number(ref self: ContractState, number: felt252) -> felt252 {
+            let new_number = self.number.read() + number;
+            self.number.write(new_number);
+            new_number
+        }
 
-    #[external(v0)]
-    fn get_number(self: @ContractState) -> felt252 {
-        self.number.read()
+        #[abi(embed_v0)]
```

Comment:
That's strange 🤔 I thought abi should be defined only once, above the trait

Reply:
Left over by accident probably, will recheck it